projects/CarouselExperiments/app/src/main/java/androidx/constraintlayout/experiments/Loader.java

```diff
@@ -69,6 +69,13 @@ static void normalMenuStartUp(MainActivity mainActivity) {
             linearLayout.addView(button);
             button.setOnClickListener(view -> launch(mainActivity, (String) view.getTag()));
         }
+        Button button = new Button(mainActivity);
+        button.setText("Carousel color demo");
+        linearLayout.addView(button);
+        button.setOnClickListener(v -> {
+            Intent intent = new Intent(mainActivity, CarouselHelperActivity.class);
+            mainActivity.startActivity(intent);
+        });
```

Comment:
Might be worth instead to modify the Loader.normalMenuStartUp to get an array of activity classes as a parameter instead, and then add them? Something around those lines, so that next demo that wants to be a separate activity would be easy to add.

Reply:
Refactoring done, in MainActivity you can load the activity classes with examples `activitiesDemo`

Reply:
Refactoring done, in MainActivity you can load the activity classes with examples activitiesDemo